### PR TITLE
adds dragStartBehavior and hitTestBehavior

### PR DESCRIPTION
**Pull request by MaddinMade**
This pull request adds the possibility to customize the DragStartBehavior and HitBoxBehavior of the underlying GestureDetector.

### CODE REVIEW

**Comment by MaddinMade** [comment]
I also implemented an animation for DragStartBehavior.down that makes it look "smoother", but that feature is probably better suited in another pull request.

**Comment by MaddinMade** [comment]
@HeavenOSK Any news on this?

File: lib/src/swipable_stack.dart [hunk 1]
```diff
@@ -74,7 +77,7 @@ class SwipableStack extends StatefulWidget {
   /// The threshold for vertical swipes.
   final double verticalSwipeThreshold;
 
-  //The set of `SwipeDirection`s you want to detect
+  /// The set of [SwipeDirection]s you want to detect.
```
Comment (HeavenOSK) [inline lib/src/swipable_stack.dart:80]: Thank you for fixing it! 🙏

**Review by HeavenOSK — APPROVED** [review]
LGTM!! 👍
 
Sorry, too late! Thanks for improvements!

**Pull request merged**